File: data/sample/atcoder/abc105_b/C/impl_02.c
#include <stdio.h>
int main(void) {
  int n,i,j;
  scanf("%d%d",&n);
  for (i = 0; i <= 15; i++) {
    for (j = 0; j <= 25; j++) {
      if (i*7 + j*4 == n) {
        printf("Yes\n");
        return (0);
      }
    }
  }
  printf("No\n");
  return (0);
}

File: data/sample/atcoder/abc105_b/Rust/impl_02.rs
use proconio::input;

fn main() {
    input! {
        n: i32,
    }
    let mut is_found = false;
    for i in 0..=n {
        let rest = n - 7 * i;
        if rest >= 0 && rest % 4 == 0 {
            is_found = true;
            break;
        }
    }
    if is_found {
        println!("Yes");
    } else {
        println!("No");
    }
}
